### PR TITLE
VI print police copy and submit

### PR DESCRIPTION
**Pull request by JustinMacaulay**
This adds in the functionality to view police copy of the VI form, and submission of the VI form. Saves the event data and creates a PDF drom the form image and saves it to the Minio bucket as an encrypted file.

### CODE REVIEW

File: python/prohibition_web_svc/middleware/event_middleware.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            encoded_pdf_filepath = f'/tmp/{encoded_file_name}'
+            with open(encoded_pdf_filepath, 'rb') as file_data:
+                client.fput_object(Config.STORAGE_BUCKET_NAME, encoded_file_name, encoded_pdf_filepath)
+            logging.debug('File uploaded')
+            
+            form_storage = FormStorageRefs(
+                form_id_vi=event.vi_form.form_id,
+                event_id=event.event_id,
+                form_type='VI',
+                storage_key=f'test/{encoded_file_name}',
```
Comment (amlanc1) [inline python/prohibition_web_svc/middleware/event_middleware.py:167]: the folder name should be read from an env variable. There is already an env variable passed on line 160. Thats the folder name (bucket name)

**Review by amlanc1 — CHANGES_REQUESTED** [review]
Updated comments

File: python/prohibition_web_svc/middleware/event_middleware.py (hunk 1 above)
Comment (JustinMacaulay) [reply python/prohibition_web_svc/middleware/event_middleware.py]: changed to be the same as the config bucket

**Review by amlanc1 — APPROVED** [review]
LGTM

**Pull request merged**